dvm-helper/dvm-helper.amd64.go

```diff
@@ -3,3 +3,4 @@
 package main
 
 const dockerArch string = "x86_64"
+const dvmArch string = "x86_64"
```

Comment:
At one point they were different but after our changes from Friday, they are the same again. I'll go with `unameArch`.